---
title: "Final Report: The Effects of Neutering on Female and Male Adult Cats"
author: "Chelsea Noack"
date: "November 29, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction
REFERENCE: 2-3 pages will be 1000-1500

The glorification of fat cats, cats which are overweight or obese, is an attraction rather than a concern in our culture. For what we often mistake as chubby cuteness is a danger to our furry friends. Cats whom are obese are at risk for diabetes type 2, lameness, oral cavity disease, urinary tract disease, neoplasia, and more [@takosova2016feline]. In addition, we find that obesity is one of the most predominant nutritional disorders within cats, staggeringly "between 11.5% and 63% [of] cats [being] overweight or obese" [@tarkosova2016feline]. The causes of obesity are, unfortunately, difficult to pinpoint. Diet is a factor, as well as genetics, environment, and overall physiologies. But is neutering a cause? Previous studies indicate that neutering may also play a factor into the weight gain of cats [@kanchuk2003weight], however the reason as to *why* reamins unclear. 

Neutering itself is proven to be important for cat population control. 

I use data from the study titled *Effects of obesity, energy restriction and neutering on the faecal microbiota of cats* by Manuela M. Fischer, et al. in order to better understand the implications behind neutering and cat weight. I specify a particular interest in sex and hypothesize that female cats will tend to be larger than male cats, especially when neutered. This is due to the differing sex hormone path which, in effect, alters the gut microbiota. To further support my analysis, I observe correlations between weight gain and sex to abundances of bacterial phylas from feline fecal samples and the age of the cats.

A study done by Kanchuk et al. suggests that "A stair-step increase in energy intake, and not decreased energy expenditure, appears to drive the weight gain associated with gonadectomy" [@kanchuk2003weight].

This study is important because humans share microbiomes with their furry friends. [@song2013cohabiting]

# Methods

## Study design

Fischer et al. used twenty-four cats from the University of California, Davis with a median age of 6.4 years. All cats were pathogen free domestic shorthairs. From the twenty-four cats, eight of the cats were obese with four males and four females. Eight cats were lean and not neutered, again with four males and four females. Eight cats, six males and two females, were lean and neutered [@fischer2017effects]. All twenty-four cats had the same diet before and during the study. Every cat underwent a physical and blood sample before the experiment began.

Cats which were lean and both neutered or not neutered housed together. All had the same diet *but* their intake was not measured. The night before the fecal sampling, the cats slept alone. Fischer et al. performed BCS (body composition scores) for each cat before sampling. 

The obese, neutered cats underwent castration/spaying 1-6 years before the study [@fischer2017effects]. Unlike the lean cats, the obese cats slept alone. The obese neutered ate the normal diet given to all cats for ten days. For six weeks, they ate 60-70% of the prior intake. Again, unlike the lean cats, their intake was now measured. Fischer et al. recorded the cats' weights every week. They also recorded fecal samples before and after the dietary restriction began [@fischer2017effects].

## Sample origin and sequencing

Before sequencing the data, Fischer et al. collected fecal samples from the cat litter box once a day for three days straight. They monitored the cats and they did not collect any samples which were 15 minutes old. To get the bacterial DNA, Fischer et al. used bead-beating methods. Bead-beating is a way to release DNA, RNA, and proteins in the cells of a sample. They used a commercial DNA extraction kit [@fischer2017effects]. To bead-beat, samples go in tubes then on a homogenizer. A homogenizer which is a lab tool used to mix materials using high energy. They mixed "for 60 seconds at a speed of 4 m/s" [@fischer2017effects].

The 16S rRNA amplified using a universal bacterial primer. A universal bacterial primers is important to interpret PCR results. Especially for microbial communities [@mao2012coverage]. PCR (polymerase chain reaction)  is a technique used to make a lot of copies of a segment of DNA. 

After PCR amplification, Fischer et al. added amplicon products of different samples. They purified the samples using AgencourtAmpure beads. The beads were important for high-throughput purification of the PCR amplicons. Then the samples combined in equal concentrations. Fischer et al. sequenced th samples using Roche 454 FLX titanium instruments [@fischer2017effects]. Pyrosequencing using instruments like Roche 454 have big advantages versus traditional Sanger sequencing. Primarily because they can examine many samples from many microbial communities [@tamaki2011analysis]. Once the raw sequences were complete, I downloaded them for my analysis. I downloaded them from NCBIâ€™s Sequence Read Archive (http://www.ncbi.nlm.nih.gov/Traces/sra) under accession no. SRP066010. 

## Computational

First, I downloaded the data from the provided link. This was a two-part process. First, I had to download the SRAtoolkit from NCBI's website. Because the data I used was in .sra format, the toolkit converts it to .fastq. I wanted .fastq so I could perform a quality check later on. After downloading the toolkit, I performed a check to make sure it worked. The steps taken came from the toolkit's website (https://www.ncbi.nlm.nih.gov/books/NBK158900/). Once I assured that it worked, I wrote a Bash script which downloaded the data into my computer. Because the files were fairly small, this did not take much time. The files downloaded into my raw data folder and saved as fastq, thanks to the toolkit. 

With the data downloaded into my computer, I transferred it to RStudio to begin my analysis. In order to do so, I downloaded necessary, but helpful, libraries. The libraries included dplyr, tidyr, knitr, ggplot2, citr, seqinr, mctoolsr, dada2, and phyloseq. Dplyr, tidyr, knitr, and ggplot2 are general use packages to help analyze the data. 

DADA2 and phyloseq were extremely important for this process. DADA2 comes from bioconductor. Bioconductor serves as the primary package to clean up and analyze our data. DADA2 specifically cleans up amplicon errors [@callahan2016]. It can also extract sample names from their original .fastq names. Phyloseq takes on microbiome analysis in R [@mcmurdie2013]. Once all files downloaded and organized, I did a QC (quality check) report on each individual file. 

Initially, I trimmed the files based off of only the QC scores (they dropped around ~400 bp). After later reviewing the amount of errors in the sequences, I decided to extend the trim to 600. To counterbalance this, I made sure the quality cut off was at a smaller number (originally at 5, now at 2). Thus, we would get more data to analyze with adequate quality. After a proper trim and error check, I deleted any duplicate sequences. 454 Roche technology sequenced the data, so I used the suggested parameters based off of (https://benjjneb.github.io/dada2/). This is when DADA2 proved to be vital, as it facilitated the process for my specific amplicon data set. In the final steps, I observed the distribution of trimmed and denoised sequences. While the data was poorly sequenced, for my purposes it would work alright. Then, I removed chimeras. 

Now, the data given is fully processed. Before analysis, I created a metadata table and a melted table to observe during research. 

# Results

I began my analysis using a grid plot in figure 1 of the overall abundance of the four sample groups between males and females. In the figure, we see that males who are neutered are in the highest abundance, followed by males who are obese undergoing weight loss and females who are obese undergoing weight loss. In the least abundance is males who are intact, obese males, and females who are intact. Given this information, I went forward and created a density plot of the overall abundance of total mass between the four sample groups. While this did not include sex, it indicated that, generally, cats obese undergoing weight loss were the most dense. The second-most dense were the obese cats. The least dense were intact cats whom kept a generally uniform distribution of total mass. That cats who were neutered were neither most or least dense, but showed the greatest range of total body mass.

**The role of body mass**

After analyzing the abundance of different phenotypes by sex and the distribution of body masses, I wanted to look deeper into the body mass question. What were the mean body masses among the four phenotypes for both male and female cats? In Table 1, the phenotype with the highest mean body mass are the obese undergoing weight loss female cats, followed by obese male cats undergoing weight loss. The phenotype with the lowest mean body mass are the intact male cats, followed by intact females. The medians of body mass are obese male cats and neutered female cats. Given the information in Table 1, I wanted to see it visually. Figure 3 depicts Table 1, and in it we see that the difference between females and males in terms of body mass to phenotype is not very significant. The most difference is between intact females and males. The obese cats undergoing weight loss are nearly similar between males and females. We see the general trend of increased body mass and phenotype along both sexes with males being generally larger. 

**The role of microbial communities**

Next, I wanted to see which microbial communities were playing a role, if at all. Thus, I made a density plot indicating the abundance of certain microbial communities in different phenotypes and sexes. Figure 4 indicates the lower range of body masses from the data, while Figure 5 shows the upper range. I divided these graphs for better visualization of the data as there were many data points to take into account. In Figure 4, we see that female cats dominate the lower weight range class among all phenotypes, showing a fairly normal distribution. Firmicutes shows the highest abundance in the lower weight ranges of both males and females. In Figure 5, males dominate the upper body weight range with uneven peaks, while females have a moderate range with only two dominant peaks. Firmicutes, again, have the highest density of both males and females. Visibly, Bacteriodetes also have a high density between the two sexes. Proteobacteria and Fusobacteria have intense fluctuations between sex and body mass. 

The microbial community question continued to linger, so I dug further. In Figure 6, I used a grid plot instead for better visualization of my question. We see a new phylum option as 'not applicable' having the highest abundance of female and male cats. After, Firmicutes, Bacteriodetes, and Actinobacteria. The lowest abundance is Fusobacteria, Proteobacteria, and Tenericutes.

**The role of age**

Lastly, I wanted to see if age played a role between the sexes when it came to remaining obese. In Figure 7, we use the data from Table 2 to see the mean ages across the phenotypes. In Table 2, we see the neutered females, and obese (with or without weight loss) females are the oldest. The neutered males, intact males, and obese males are the youngest. In Figure 7, we see females in all phenotypes have the larger mean age groups while males dominate the younger age groups. The intact cats have similar mean age patterns between males and females. The neutered, obese, and obese undergoing weight loss phenotypes show the strongest variance between the sexes. Next, I wanted to find out the mean sex across the bacteria phylas and cat phenotypes for females and males. This would be a far more hefty task. In Table 3, we see that females are still, on average, older. Intact phenotypes have considerably less bacteria phyla indicated here than the other phenotypes. Females on average are still older than males. I wanted to visualize the data in the best way possible, so a bar graph seemed like an effective solution. In Figure 8, we see the intact phenotype has three fewer bacterial phyla than the other phenotypes. Neutered phenotypes have older ages, on average. Obese and obese undergoing weight loss typically have similar ages as well as bacterial phylas. 

```{r load-libraries, echo=FALSE, message = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")

# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")

# This to export a fasta of our final denoised sequence variants
library("seqinr")

# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")

# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")

# Install devtools package
devtools::install_github("ropenscilabs/gramr")
```

```{r}
load("output/phyloseqobject.RData")
```

```{r meltedtable, echo=FALSE}
# Compile into a table
melted_obj <- psmelt(phyloseq_obj)
```

```{r gridplot-sex-abundance-phenotype}
# First figure indicating the general schema of the data
ggplot(data=melted_obj, 
          aes(x=host_sex_s, y=Abundance)) +
  geom_point() +
  facet_grid(. ~ host_phenotype_s) +
  labs(title = "Abundance of male and female cats with different phenotypes",
       x = "Sex of Cat")
```

**Figure 1:** A grid plot of abundance of female and male cats whom are intact, neutered, obese, and who were obese before weight loss. 

```{r density-mass-phenotype}
# Second figure using density to show the range of body mass to phenotype
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_phenotype_s)) +
  geom_density(alpha = 0.1) +
  xlim(3400, 7000) +
  labs(title = "Distribution of body size in different phenotypes",
       x = "Total Mass of Cats", 
       y = "Density", 
       fill = "Cat Phenotype")
```

**Figure 2**: A density plot of host's total mass to its phenotype. 

```{r make-table, echo = FALSE}
# Comment here
meanmass_8 <- melted_obj %>%
  group_by(host_phenotype_s, host_sex_s) %>%
  summarize(mean_bodymass = mean(host_tot_mass_s)) %>%
  arrange(desc(host_phenotype_s)) %>%
  head(n = 8)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
meanmass_8 %>%
  kable()
```

**Table 1**: Mean body mass among phenotypes and sexes. 

```{r}
meanmass_8 %>%
  ggplot(aes(x = host_sex_s,
             y = mean_bodymass,
             fill = host_phenotype_s)) +
    geom_col(position = "dodge") +
  ggtitle("Cat sex to mean body mass over four phenotypes") +
  xlab("Cat Sex") + ylab("Mean Body Mass") + labs(fill = "Cat Phenotype")
```

**Figure 3**: Using the table, the mean body mass visualized with a bar graph. 

```{r density-sex-lowermass-phylum}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_sex_s, colour = Phylum)) +
  geom_density(alpha = 0.1) +
  xlim(2500, 5000) +
  ggtitle("Lower distribution of body mass across bacterial phyla and cat sex") +
  xlab("Cat total mass") + ylab("Density") + labs(fill = "Cat sex", colour = "Bacteria phylum")
```

**Figure 4**: A density plot of the lower range weight class of females and males and the correlating Phylums. 

```{r density-sex-uppermass-pylum}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_sex_s, colour = Phylum)) +
  geom_density(alpha = 0.1) +
  xlim(5500, 7000) +
    ggtitle("Upper distribution of body mass across bacterial phyla and cat sex") +
  xlab("Cat total mass") + ylab("Density") + labs(fill = "Cat sex", colour = "Bacteria phylum")
```

**Figure 5**: A density plot of the upper range weight class of females and males and the correlating Phylums. 

```{r gridplot-abundance-sex-phylum}
# Comment here
ggplot(data=melted_obj, 
          aes(x=host_sex_s, y=Abundance)) +
  geom_point() +
  facet_grid(. ~ Phylum) +
  labs(title = "Abundance of male and female cats to bacterial phylum",
       x = "Sex of Cat")
```

**Figure 6**: A grid plot of the abundance of bacterial phylums in both males and females. 

```{r make-table, echo = FALSE}
# Comment here
meanage_10 <- melted_obj %>%
  group_by(host_phenotype_s, host_sex_s) %>%
  summarize(mean_age = mean(host_age_s)) %>%
  arrange(desc(host_phenotype_s)) %>%
  head(n = 10)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
meanage_10 %>%
  kable()
```

**Table 2**: A table of the mean ages for each phenotype in male and female cats. 

```{r baplot-phenotype-sex-age, echo = FALSE}
# Comment here
meanage_10 %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_age,
             fill = host_sex_s)) +
    geom_col(position = "dodge") +
  labs(title = "Male and female phenotypes measured to mean age",
       x = "Cat Phenotype",
       y = "Cat Age",
       fill = "Cat Sex")
```

**Figure 7**: A bar plot indicating different mean ages between the categories intact, neutered, obese, and obese after weight loss among females and males. 

```{r make-table, echo = FALSE}
# Comment here
meanaps_30 <- melted_obj %>%
  group_by(host_phenotype_s, Phylum, host_sex_s) %>%
  summarize(mean_agephylumsex = mean(host_age_s)) %>%
  arrange(desc(host_phenotype_s)) %>%
  head(n = 50)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
meanaps_30 %>%
  kable()
```

**Table 3**: A table indicating mean ages for the different cat phenotypes, bacteria phylas, and sex. 

```{r barplot-age-phenotype-phylum, echo = FALSE}
# Comment here
meanaps_30 %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_agephylumsex,
             fill = Phylum)) +
    geom_col(position = "dodge") +
    labs(title = "Phenotypes and bacterial phyla measured to mean age",
       x = "Cat Phenotype",
       y = "Cat Age",
       fill = "Phylum")
```

**Figure 8**: A bar plot showing age against phylum and category of intact, neutered, obese, and obese after weight loss. 

# Discussion

My hypothesis was that female cats would be more obese than male cats, especially after being neutered. However, the data indicates opposite results. Male cats are heavier than female cats within all phenotypes; intact, neutered, obese, and obese undergoing weight loss. We see more males who are neutered due to sample design. 

**The role of body mass**

Female obese cats (undergoing and not undergoing weight loss) were *not* heavier than the male cats. In table 1, we see that within every phenotype the male cats are more obese than the female cats. Within the obese phenotype, the males and females were fairly similar. But we see that even after the weight loss treatment, males are significantly larger than the females and actually retain almost the same average as the obese cats. Intact female cats are lighter than neutered. The same goes for the males. Intact males are lighter than neutered males. So, there appears to be a pattern, but we cannot say this is always the case. It only pertains to this study. Additionally, obese females and males were nearly the same weight range, but males had a harder time losing weight. Why? Perhaps the biochemical pathways or lipid retention is different in males than in females.The lowest weight was 3.7 kg female intact cat. The highest weight was an obese 6.7 kg male cat. We see that, again, males are heavier than females, which counters my hypothesis. "Feline obesity is associated with a significant increase in FT4 (free serum throxine) within the normal range" [@ferguson2007obesity]. 

**The role of microbial communities**

Firmicutes made an appearance across the lower body mass range and upper body mass range. Admittedly, the mammalian GI tract is highly diverse [@suchodolski2011intestinal], thus looking for a single community of bacteria possibly affiliated with obesity and neutering is similar to finding a needle in a haystack. Firmicutes did appear most frequently within all body mass sizes, which indicates to me that the bacteria is *not* dependent on a weight or neutering factor, or atleast is not explicity correlated. The overabundance of Firmicutes and Bacteriodetes is consistent with previous studies about feline fecal microbial communities [@ritchie2010characterization]. 

What perhaps is more proactive is not looking at the bacteria phyla which appeared in both males, females, and phenotypes, but observing the ones which only occured in a few. In 

**The role of age**

Age has been shown to alter the microbiota community within humans [@mariat2009firmicutes]. Is that the same for felines? If so, would it also be affected by neutering and weight gain/maintenance? These were all questions I had as I approached the final aspect of my analysis. 

Typically, felines undergo neutering early in life [@deusch2015longitudinal]. In a study done by Deusch et al., they wanted to identify the connection between We conclude that the feline faecal microbiome is predominantly determined by age when diet and environment are controlled for. 

**Alternative Conclusions**

If sex does not factor into the weight gain or loss in cats, what else could? Underlying my hypothesis was the presumption that the male and female hormones were explicit. However, it might be the hormones working in tandem with other pathways. In a study done by Hoenig et al., they found that "changes in NEFA suppression, caloric intake, and leptin concentrations may be indicators of, and possible risk factors for, the development of obesity in cats after neutering" [@hoenig2002effects]. 

**Future Directions**

REFERENCE: 1500-2000 words



# Sources Cited
