---
title: "Final Report: Your Title Here"
author: "Don Francisco"
date: "November 22, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction
REFERENCE: 2-3 pages will be 1000-1500

- Domestication of cats
- Our love for cats
- Being obese, as well as for humans, will cause early death, include statistics
- What if it's not just diet? Does neutering play a role? Population control is important too. Here I ask if there are particular bacterias within striclty obese, neutered cats that is preventing them from maintaining a healthy weight? If so, can this lead to a positive medication which can be administered to the cats?
- In the study by so and so, they studied intact and neutered cats in order to find a connection between obesity and being neutered.

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design
REFERENCE: 250 words = 1/2 page

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Sample origin and sequencing
REFERENCE: 250 words = 1/2 page

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Computational
REFERENCE: 250-500 words

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r load-libraries, echo=FALSE, message = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")

# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")

# This to export a fasta of our final denoised sequence variants
library("seqinr")

# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")

# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")

# Install devtools package
devtools::install_github("ropenscilabs/gramr")
```

```{r}
load("output/phyloseqobject.RData")
```

```{r meltedtable, echo=FALSE}
# Compile into a table
melted_obj <- psmelt(phyloseq_obj)
```

```{r gridplot-sex-abundance-phenotype}
# COMMENT HERE
ggplot(data=melted_obj, 
          aes(x=host_sex_s, y=Abundance)) +
  geom_point() +
  facet_grid(. ~ host_phenotype_s)
```

**Figure 1:** A grid plot of abundance of female and male cats whom are intact, neutered, obese, and who were obese before weight loss.

```{r gridplot-abundance-sex-phylum}
# Comment here
ggplot(data=melted_obj, 
          aes(x=host_sex_s, y=Abundance)) +
  geom_point() +
  facet_grid(. ~ Phylum)
```

**Figure 2**: A grid plot of the abundance of certain phylums in both males and females.

```{r baplot-phenotype-sex-age, echo = FALSE}
# Comment here
melted_obj %>%
  ggplot(aes(x = host_phenotype_s,
             y = host_age_s,
             fill = host_sex_s)) +
    geom_col(position = "dodge")
```

**Figure 3**: A bar plot indicating different ages between the categories intact, neutered, obese, and obese after weight loss among females and males.

```{r barplot-age-phenotype-phylum, echo = FALSE}
# Comment here
melted_obj %>%
  ggplot(aes(x = host_phenotype_s,
             y = host_age_s,
             fill = Phylum)) +
    geom_col(position = "dodge")
```

**Figure 4**: A bar plot showing age against phylum and category of intact, neutered, obese, and obese after weight loss.

```{r density-mass-phenotype}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_phenotype_s, colour = host_phenotype_s)) +
  geom_density(alpha = 0.1) +
  xlim(3400, 7000)
```

**Figure 5**: A density plot of host's total mass to its phenotype. 

```{r density-sex-lowermass-phylum}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_sex_s, colour = Phylum)) +
  geom_density(alpha = 0.1) +
  xlim(2500, 5000)
```

**Figure 6**: A density plot of the lower range weight class of females and males and the correlating Phylums.

```{r density-sex-uppermass-pylum}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_sex_s, colour = Phylum)) +
  geom_density(alpha = 0.1) +
  xlim(5500, 7000)
```

**Figure 7**: A density plot of the upper range weight class of females and males and the correlating Phylums.

In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion
REFERENCE: 1500-2000 words

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
