---
title: "Final Report: The Effects of Neutering on Female and Male Adult Cats"
author: "Chelsea Noack"
date: "November 29, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

REFERENCE: 2-3 pages will be 1000-1500

- Domestication of cats
- Our love for cats
- Being obese, as well as for humans, will cause early death, include statistics
- **HYPOTHESIS**: Females will be larger overall, especially after neutering, than male cats. This puts female cats at a severe disadavntage. 
- What if it's not just diet? Does neutering play a role? Population control is important too. Here I ask if there are particular bacterias within striclty obese, neutered cats that is preventing them from maintaining a healthy weight? If so, can this lead to a positive medication which can be administered to the cats?
- In the study by so and so, they studied intact and neutered cats in order to find a connection between obesity and being neutered.

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

Fischer et al. used twenty-four cats from the University of California, Davis with a median age of 6.4 years. All cats were pathogen free domestic shorthairs. From the twenty-four cats, eight of the cats were obese with four males and four females. Eight cats were lean and not neutered, again with four males and four females. Eight cats, six males and two females, were lean and neutered [@fischer2017effects]. All twenty-four cats had the same diet before and during the study. Every cat underwent a physical and blood sample before the experiment began.

Cats which were lean and both neutered or not neutered housed together. All had the same diet *but* their intake was not measured. The night before the fecal sampling, the cats slept alone. Fischer et al. performed BCS (body composition scores) for each cat before sampling. 

The obese, neutered cats underwent castration/spaying 1-6 years before the study [@fischer2017effects]. Unlike the lean cats, the obese cats slept alone. The obese neutered ate the normal diet given to all cats for ten days. For six weeks, they ate 60-70% of the prior intake. Again, unlike the lean cats, their intake was now measured. Fischer et al. recorded the cats' weights every week. They also recorded fecal samples before and after the dietary restriction began [@fischer2017effects].

## Sample origin and sequencing

Before sequencing the data, Fischer et al. collected fecal samples from the cat litter box once a day for three days straight. They monitored the cats and they did not collect any samples which were 15 minutes old. To get the bacterial DNA, Fischer et al. used bead-beating methods. Bead-beating is a way to release DNA, RNA, and proteins in the cells of a sample. They used a commercial DNA extraction kit [@fischer2017effects]. To bead-beat, samples go in tubes then on a homogenizer. A homogenizer which is a lab tool used to mix materials using high energy. They mixed "for 60 seconds at a speed of 4 m/s" [@fischer2017effects].

The 16S rRNA amplified using a universal bacterial primer. A universal bacterial primers is important to interpret PCR results. Especially for microbial communities [@mao2012coverage]. PCR (polymerase chain reaction)  is a technique used to make a lot of copies of a segment of DNA. 

After PCR amplification, Fischer et al. added amplicon products of different samples. They purified the samples using AgencourtAmpure beads. The beads were important for high-throughput purification of the PCR amplicons. Then the samples combined in equal concentrations. Fischer et al. sequenced th samples using Roche 454 FLX titanium instruments [@fischer2017effects]. Pyrosequencing using instruments like Roche 454 have big advantages versus traditional Sanger sequencing. Primarily because they can examine many samples from many microbial communities [@tamaki2011analysis]. Once the raw sequences were complete, I downloaded them for my analysis. I downloaded them from NCBIâ€™s Sequence Read Archive (http://www.ncbi.nlm.nih.gov/Traces/sra) under accession no. SRP066010. 

## Computational

First, I downloaded the data from the provided link. This was a two-part process. First, I had to download the SRAtoolkit from NCBI's website. Because the data I used was in .sra format, the toolkit converts it to .fastq. I wanted .fastq so I could perform a quality check later on. After downloading the toolkit, I performed a check to make sure it worked. The steps taken came from the toolkit's website (https://www.ncbi.nlm.nih.gov/books/NBK158900/). Once I assured that it worked, I wrote a Bash script which downloaded the data into my computer. Because the files were fairly small, this did not take much time. The files downloaded into my raw data folder and saved as fastq, thanks to the toolkit. 

With the data downloaded into my computer, I transferred it to RStudio to begin my analysis. In order to do so, I downloaded necessary, but helpful, libraries. The libraries included dplyr, tidyr, knitr, ggplot2, citr, seqinr, mctoolsr, dada2, and phyloseq. Dplyr, tidyr, knitr, and ggplot2 are general use packages to help analyze the data. 

DADA2 and phyloseq were extremely important for this process. DADA2 comes from bioconductor. Bioconductor serves as the primary package to clean up and analyze our data. DADA2 specifically cleans up amplicon errors [@callahan2016]. It can also extract sample names from their original .fastq names. Phyloseq takes on microbiome analysis in R [@mcmurdie2013]. Once all files downloaded and organized, I did a QC (quality check) report on each individual file. 

Initially, I trimmed the files based off of only the QC scores (they dropped around ~400 bp). After later reviewing the amount of errors in the sequences, I decided to extend the trim to 600. To counterbalance this, I made sure the quality cut off was at a smaller number (originally at 5, now at 2). Thus, we would get more data to analyze with adequate quality. After a proper trim and error check, I deleted any duplicate sequences. 454 Roche technology sequenced the data, so I used the suggested parameters based off of (https://benjjneb.github.io/dada2/). This is when DADA2 proved to be vital, as it facilitated the process for my specific amplicon data set. In the final steps, I observed the distribution of trimmed and denoised sequences. While the data was poorly sequenced, for my purposes it would work alright. Then, I removed chimeras. 

Now, the data given is fully processed. Before analysis, I created a metadata table and a melted table to observe during research. 

# Results

The data proved to be interesting and not perfectly correlated to my hypothesis. I began my analysis using a grid plot in figure 1 of the overall abundance of the four sample groups between males and females. In the figure, we see that males who are neutered are in the highest abundance, followed by males who are obese undergoing weight loss and females who are obese undergoing weight loss. In the least abundance is males who are intact, obese males, and females who are intact. Given this information, I went forward and created a density plot of the overall abundance of total mass between the four sample groups. While this did not include sex, it indicated that, generally, cats obese undergoing weight loss were the most dense. The second-most dense were the obese cats. The least dense were intact cats whom kept a generally uniform distribution of total mass. That cats who were neutered were neither most or least dense, but showed the greatest range of total body mass. 


## Subsections are ok in the results section too

```{r load-libraries, echo=FALSE, message = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")

# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")

# This to export a fasta of our final denoised sequence variants
library("seqinr")

# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")

# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")

# Install devtools package
devtools::install_github("ropenscilabs/gramr")
```

```{r}
load("output/phyloseqobject.RData")
```
```{r meltedtable, echo=FALSE}
# Compile into a table
melted_obj <- psmelt(phyloseq_obj)
```

```{r gridplot-sex-abundance-phenotype}
# First figure indicating the general schema of the data
ggplot(data=melted_obj, 
          aes(x=host_sex_s, y=Abundance)) +
  geom_point() +
  facet_grid(. ~ host_phenotype_s)
```

**Figure 1:** A grid plot of abundance of female and male cats whom are intact, neutered, obese, and who were obese before weight loss. 

```{r density-mass-phenotype}
# Second figure using density to show the range of body mass to phenotype
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_phenotype_s, colour = host_phenotype_s)) +
  geom_density(alpha = 0.1) +
  xlim(3400, 7000)
```

**Figure 2**: A density plot of host's total mass to its phenotype. 

```{r gridplot-abundance-sex-phylum}
# Comment here
ggplot(data=melted_obj, 
          aes(x=host_sex_s, y=Abundance)) +
  geom_point() +
  facet_grid(. ~ Phylum)
```

**Figure 2**: A grid plot of the abundance of certain phylums in both males and females.

```{r baplot-phenotype-sex-age, echo = FALSE}
# Comment here
melted_obj %>%
  ggplot(aes(x = host_phenotype_s,
             y = host_age_s,
             fill = host_sex_s)) +
    geom_col(position = "dodge")
```

**Figure 3**: A bar plot indicating different ages between the categories intact, neutered, obese, and obese after weight loss among females and males.

```{r barplot-age-phenotype-phylum, echo = FALSE}
# Comment here
melted_obj %>%
  ggplot(aes(x = host_phenotype_s,
             y = host_age_s,
             fill = Phylum)) +
    geom_col(position = "dodge")
```

**Figure 4**: A bar plot showing age against phylum and category of intact, neutered, obese, and obese after weight loss.

```{r make-table, echo = FALSE}
# Comment here
meanmass_8 <- melted_obj %>%
  group_by(host_phenotype_s, host_sex_s) %>%
  summarize(mean_bodymass = mean(host_tot_mass_s)) %>%
  arrange(desc(host_phenotype_s)) %>%
  head(n = 8)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
meanmass_8 %>%
  kable()
```

**Table 1**: Mean body mass among phenotypes and sexes. 

```{r}
meanmass_8 %>%
  ggplot(aes(x = host_sex_s,
             y = mean_bodymass,
             fill = host_phenotype_s)) +
    geom_col(position = "dodge")
```

**Figure 6**: Using the table, the mean body mass visualized. 

```{r density-sex-lowermass-phylum}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_sex_s, colour = Phylum)) +
  geom_density(alpha = 0.1) +
  xlim(2500, 5000)
```

**Figure 7**: A density plot of the lower range weight class of females and males and the correlating Phylums.

```{r density-sex-uppermass-pylum}
# Comment here
melted_obj %>%
ggplot(aes(host_tot_mass_s, fill = host_sex_s, colour = Phylum)) +
  geom_density(alpha = 0.1) +
  xlim(5500, 7000)
```

**Figure 8**: A density plot of the upper range weight class of females and males and the correlating Phylums.

CONCLUSION

# Discussion
REFERENCE: 1500-2000 words

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
